src/masonite/orm/schema/Schema.py

```diff
@@ -83,3 +83,19 @@ def has_column(cls, table, column, query_only=False):
     def set_default_string_length(cls, length):
         cls._default_string_length = length
         return cls
+
+    @classmethod
+    def drop_table_query(cls, table, query_only=True):
```

Comment:
You are right. I made the test before I made some changes. Just change this to `drop_table`